### ISSUE TITLE
[Feedback]: A soft requirement for issue response time

### ISSUE THREAD

**Issue by lonegunmanb** [Azure/Azure-Verified-Modules#88]
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Description

[ID: SNFR11 - Category: Contribution/Support - Issues Response Times](https://github.com/Azure/Azure-Verified-Modules/blob/060bab728a8b0f04666b66cd593051b2c691d811/docs/content/specs/shared/_index.md?plain=1#L403) describes `A module owner **MUST** respond to logged issues within 3 business days.`.

According to our experience in `terraform-provider-azurerm` maintenance work, it would be hard to follow this requirement, especially when the module's creators and maintainers are doing the job as a side job. This requirement could encourage a bot-style response at first time: `Thanks for opening this issue to us, we'll take a look ASAP.`.

Maybe we should have a background scan service, looking for those issues that has no response for weeks.

**Comment by jtracey93** [comment]
Hi @lonegunmanb,

We agree that this may be not always easy to achieve however, this is a requirement from our leadership, that we already successfully meet in other projects we run. So we will leave as is for now and monitor this.

We are also creating the automation to find these "breaches" and notify folks of them and this will be enabled very soon in all repos, via the template repos etc. 👍
